ngraph/core/include/ngraph/op/roi_pooling.hpp

```diff
@@ -31,42 +31,47 @@ namespace ngraph
                 const NodeTypeInfo& get_type_info() const override { return type_info; }
                 enum class ROIPoolingMethod
                 {
-                    BLN, // Bilinear interpolation
-                    MAX  // Maximum
+                    Bilinear, // Bilinear interpolation
+                    Max       // Maximum
                 };
 
                 ROIPooling() = default;
                 /// \brief Constructs a ROIPooling operation
                 ///
                 /// \param input          Input feature map {N, C, H, W}
                 /// \param coords         Coordinates of bounding boxes
-                /// \param output_size    Height/Width of ROI output features
+                /// \param pooled_h       Height of the ROI output features
+                /// \param pooled_w       Width of the ROI output features
                 /// \param spatial_scale  Ratio of input feature map over input image size
                 /// \param method         Method of pooling - Max or Bilinear
                 ROIPooling(const Output<Node>& input,
                            const Output<Node>& coords,
-                           const Shape& output_size,
+                           const int pooled_h,
```

Comment:
Changing operation semantic requires introduction of a new operation. We cannot modify the API of existing operation! Please, revert these changes.

Reply:
Done